nvtabular/ops/operator.py

```diff
@@ -13,6 +13,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-from merlin.dag import BaseOperator, ColumnSelector  # noqa pylint: disable=unused-import
+from merlin.dag import (  # noqa pylint: disable=unused-import
+    BaseOperator,
+    ColumnSelector,
+    DataFormats,
+)
 
-Operator = BaseOperator
+
+# Avoid TENSOR_TABLE by default (for now)
+class Operator(BaseOperator):
+    @property
+    def supported_formats(self):
+        return DataFormats.PANDAS_DATAFRAME | DataFormats.CUDF_DATAFRAME
```

Comment:
With https://github.com/NVIDIA-Merlin/systems/pull/389 applied to `systems`, I was seeing inference errors in `Groupby`. This is because the `Groupby` operator calls `sort_values`, and sorting is not supported for merlins `TensorTable` abstraction.

Since I am not entirely sure which NVTabular operations are supported with `TensorTable`, I figured the safest fix was to assume **none** of the operations are supported for now. (Any thoughts on this @jperez999 ?)

In a follow-up to this PR, it probably makes sense to add `GeneralOperator = BaseOperator` above this. That way, any operator with known support for `TensorTable` could inherit from `GeneralOperator` instead.

Reply:
@oliverholworthy and @jperez999 do you have a chance to look at Rick's comments above? thanks.

Reply:
This should be alright. All of the operators in nvtabular were created with data frames in mind. If we ever decide to add in tensor table, we can make the change then. If this speeds up the runs I say we should do it and solves breaking issues we should execute.